apps/framework-cli/src/project.rs

```diff
@@ -161,7 +161,7 @@ impl Project {
             .build()?
             .try_deserialize()?;
 
-        project_config.project_location = directory.clone();
+        project_config.project_location.clone_from(&directory);
```

Comment:
<img width="1184" alt="Screenshot 2024-05-02 at 11 50 32 AM" src="https://github.com/514-labs/moose/assets/7286437/32ac8b01-f0d1-4df3-885b-5457bf3674a4">

There was a clippy error on code that wasn't changed. I had to run `rustup update` locally to reproduce. https://rust-lang.github.io/rust-clippy/master/index.html#/assigning_clones